grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -917,6 +917,21 @@ class QueryService implements Status {
         (params.size() > 0) ? query + " AND ${q} " : query + q
     }
 
+    /**
+     * Find all issues
+     * @return List of Issues
+     */
+    List<Issue> findIssues() {
+        final String query = 'SELECT * FROM issue WHERE deleted = 0'
+        final session = sessionFactory.currentSession
+        final SQLQuery sqlQuery = session.createSQLQuery(query)
+        final List<Issue> results = sqlQuery.with {
+            addEntity(Issue)
```

Comment:
`addEntity` is a pretty slow Hibernate operation. You might want to consider handling the results in a more robust mechanism to convert the query responses to an `Issue` object without using ORM.